src/resources/resource_paths.c

```diff
@@ -136,6 +136,13 @@ static bool find_resource_path(path *dst) {
         }
         str_free(&tmp);
     }
+    str_from_c(&tmp, ".");
+    // Check cwd.
+    if(scan_potential_resource_dirs(dst, &tmp, "openomf/resources/openomf.bk", "openomf/")) {
+        log_debug("Resources found in system path: %s", path_c(dst));
+        goto ok;
+    }
+    str_free(&tmp);
```

Comment:
I don't think this belongs to the game binary -- it is only used by tests, afaiks there is no other purpose for this. I think the envvar is the way to go here.